.circleci/ci/src/pipelines/tests/resources/build-rpm-and-docker-images/build-rpm-and-docker-images-prerelease-dry-run.yml

```diff
@@ -74,10 +74,54 @@ jobs:
           name: Build & Publish Gravitee.io APIM Docker images
           command: |
             echo "DRY RUN Mode. Build only"
-            docker buildx build --platform=linux/arm64,linux/amd64 --build-arg GRAVITEEIO_VERSION=4.1.0-alpha.1 --build-arg GRAVITEEIO_DOWNLOAD_URL=https://gravitee-dry-releases-downloads.cellar-c2.services.clever-cloud.com/pre-releases/graviteeio-apim/distributions --quiet  -t graviteeio/apim-gateway:4.1.0-alpha.1 -t graviteeio/apim-gateway:4.1.0-alpha  -f ./gravitee-apim-gateway/docker/Dockerfile-from-download ./gravitee-apim-gateway/docker
-            docker buildx build --platform=linux/arm64,linux/amd64 --build-arg GRAVITEEIO_VERSION=4.1.0-alpha.1 --build-arg GRAVITEEIO_DOWNLOAD_URL=https://gravitee-dry-releases-downloads.cellar-c2.services.clever-cloud.com/pre-releases/graviteeio-apim/distributions --quiet  -t graviteeio/apim-management-api:4.1.0-alpha.1 -t graviteeio/apim-management-api:4.1.0-alpha  -f ./gravitee-apim-rest-api/docker/Dockerfile-from-download ./gravitee-apim-rest-api/docker
-            docker buildx build --platform=linux/arm64,linux/amd64 --build-arg GRAVITEEIO_VERSION=4.1.0-alpha.1 --build-arg GRAVITEEIO_DOWNLOAD_URL=https://gravitee-dry-releases-downloads.cellar-c2.services.clever-cloud.com/pre-releases/graviteeio-apim/distributions --quiet  -t graviteeio/apim-management-ui:4.1.0-alpha.1 -t graviteeio/apim-management-ui:4.1.0-alpha  -f ./gravitee-apim-console-webui/docker/Dockerfile-from-download ./gravitee-apim-console-webui/docker
-            docker buildx build --platform=linux/arm64,linux/amd64 --build-arg GRAVITEEIO_VERSION=4.1.0-alpha.1 --build-arg GRAVITEEIO_DOWNLOAD_URL=https://gravitee-dry-releases-downloads.cellar-c2.services.clever-cloud.com/pre-releases/graviteeio-apim/distributions --quiet  -t graviteeio/apim-portal-ui:4.1.0-alpha.1 -t graviteeio/apim-portal-ui:4.1.0-alpha  -f ./gravitee-apim-portal-webui/docker/Dockerfile-from-download ./gravitee-apim-portal-webui/docker
+            docker buildx build --platform=linux/arm64,linux/amd64 --build-arg GRAVITEEIO_VERSION=4.1.0-alpha.1 --build-arg GRAVITEEIO_DOWNLOAD_URL=https://gravitee-dry-releases-downloads.cellar-c2.services.clever-cloud.com/pre-releases/graviteeio-apim/distributions --quiet -t graviteeio/apim-gateway:4.1.0-alpha.1 -t graviteeio/apim-gateway:4.1.0-alpha  -f ./gravitee-apim-gateway/docker/Dockerfile-from-download ./gravitee-apim-gateway/docker
```

Comment:
looks a bit weird to me to have the version 4.1.0-alpha but it is not related to your PR directly :)

Reply:
I'm not sure to understand, it is for when we release "pre-release" version of the product 🙈